packages/web-app-files/src/views/shares/SharedViaLink.vue

```diff
@@ -112,6 +113,12 @@ export default defineComponent({
     const { loadResourcesTask, selectedResourcesIds, paginatedResources } =
       useResourcesViewDefaults<Resource, any, any[]>()
 
+    const disabledResources = computed(() => {
```

Comment:
is it possible to refactor this into a composable since its used multiple times?

Reply:
That's a great idea !